sql/000_001_006-name_cleaning.sql

```diff
@@ -0,0 +1,130 @@
+/* this file will create a table defining substituations for simplifying */
+/* company names in preparation for fuzzy matching */ 
+
+CREATE TABLE etl.company_name_abbreviations (
+  to_replace TEXT NOT NULL UNIQUE,
+  replacement TEXT NOT NULL,
+  regexp_flags TEXT NOT NULL DEFAULT 'ig'
+);
+CREATE INDEX ix_company_name_replacement_cover ON etl.company_name_abbreviations (
+  to_replace,
+  replacement,
+  regexp_flags
+);
+
+INSERT INTO etl.company_name_abbreviations (
+  to_replace,
+  replacement,
+  regexp_flags
+) VALUES
+('\s+and\s+', ' & ', 'ig'),
+('\s+en\s+', ' & ', 'ig'),
+('\s+och\s+', ' & ', 'ig'),
+('\s+und\s+', ' & ', 'ig'),
+('\(inactive\)', '', 'ig'),
+('aktg', 'ag', 'ig'),
+('aktiengesellschaft', 'ag', 'ig'),
+/* covers associate AND associates, single and plural */
+('associate[s]*', 'assoc', 'ig'),
+('berhad', 'bhd', 'ig'),
+('company', 'co', 'ig'),
+('corporation', 'corp', 'ig'),
+('designated\s+activity[\s\$]+co', 'dac', 'ig'),
+('develop(ment){0,1}', 'dev', 'ig'),
+('financial', 'fin', 'ig'),
+('generation', 'gen', 'ig'),
+('government', 'govt', 'ig'),
+('group', 'grp', 'ig'),
+('holding(s){0,1}', 'hldgs', 'ig'),
+('incorporated', 'inc', 'ig'),
+('international', 'intl', 'ig'),
+('investment', 'invest', 'ig'),
+/* lp: limited -> ltd AND partnership -> prthip, so this turns variants into lp */
+('ltd\s+prthip', 'lp', 'ig'),
+('limited', 'ltd', 'ig'),
+('ltd\s+liability[\s\$]+co', 'llc', 'ig'),
+('partner(s){0,1}', 'prt', 'ig'),
+/* plc: limited -> ltd AND company -> co, so this turns variants into plc */
+('public\s+ltd[\s\$]+co', 'plc', 'ig'),
```

Comment:
Do we want a public -> pub rule? If you didn't see any examples of that in your analysis than probably not worth adding

Reply:
I didn't do much analysis on the names this time around. This PR is just the replacements more or less as they are in `DataStore`